src/Vogen/Templates/Byte/Byte_SystemTextJsonConverter.cs

```diff
@@ -3,13 +3,11 @@ class VOTYPESystemTextJsonConverter : global::System.Text.Json.Serialization.Jso
         {
             public override VOTYPE Read(ref global::System.Text.Json.Utf8JsonReader reader, global::System.Type typeToConvert, global::System.Text.Json.JsonSerializerOptions options)
             {
-__NORMAL__                return new VOTYPE(reader.GetByte());
-__STRING__                return new VOTYPE(global::System.Byte.Parse(reader.GetString(), global::System.Globalization.NumberStyles.Any, global::System.Globalization.CultureInfo.InvariantCulture));
+                return VOTYPE.From(reader.GetByte());
```

Comment:
This should've broken some tests. The old code had placeholder tokens for a recent addition that allowed numbers to be serialized as strings